pkg/server/container_create_test.go

```diff
@@ -177,13 +177,15 @@ func TestGeneralContainerSpec(t *testing.T) {
 	testPid := uint32(1234)
 	config, sandboxConfig, imageConfig, specCheck := getCreateContainerTestData()
 	c := newTestCRIContainerdService()
-	spec, err := c.generateContainerSpec(testID, testPid, config, sandboxConfig, imageConfig, nil)
+	testSandboxID := "SandboxID"
```

Comment:
You may want to check whether the annotations are set properly in default `specCheck`. However, if you don't have to do that, I can send a following up PR to do that. :)

Reply:
yeah agree - not worries, adding proper checks to specCheck